statics/js/components/preference.js

```diff
@@ -0,0 +1,133 @@
+/* jshint multistr: true */
+
+var PreferenceComponent = {
+
+  name: 'Preference',
+
+  mixins: [notificationMixin],
+
+  template: '\
+    <form class="form-preference" @submit.prevent="save">\
+      <div class="form-group">\
```

Comment:
maybe remove this one as not used yet or not working

Reply:
removed